win/first-run.bat

```diff
@@ -18,6 +21,10 @@ IF NOT EXIST %AppData%\Hush\hush.conf (
 
 IF NOT EXIST %LocalAppData%\HushSwingWalletUI\addressBook.csv (
    (
-    echo t1dSqorCh1PdyvViTRHzKdAFjnR81VyGv1u,HUSH Dev Fund
+    echo t1h6kmaQwcuyejDLazT3TNZfV8EEtCzHRhc,HUSH Donation address
+	echo t1dxPQTkv3j6ifqRKQRkMpBrDyhKWm3WXma, TheTrunk
```

Comment:
I don't think including individual dev addresses in the wallet addressbook bootstrap is beneficial to the project, the readme would be a better spot for these

Dev team address is fine

Reply:
Agree